### ISSUE TITLE
Focused Launch: Plans grid in Plan details view does not highlight the selected plan

### ISSUE THREAD

**Issue by ciampo** [Automattic/wp-calypso#47814]
## What

Currently, the Plans grid used in the Plan details view in the Focused Launch modal doesn't highlight the plan currently selected by the user.

This is in contrast with the rest of the Focused Launch UI, where the user can always visually understand what are the currently selected domain and plan.

<img width="1027" alt="Screenshot 2020-11-26 at 23 05 04" src="https://user-images.githubusercontent.com/1083581/100393985-d46c4080-303b-11eb-8b69-b6256e4c2289.png">

## Why

When the user navigates to the Plan details view, they may feel disoriented by seeing the plan that they have selected not being highlighted.

## Acceptance Criteria

The button of the currently selected plan in the Plans grid should:
- [ ] Read " ✔️ Current Selection"
- [ ] Use `#FAFCFE` as the background color (we should try to use a pre-existing color variable, if there's one that matches)
- [ ] Use the primary blue color for borders and text
- [ ] The 2 changes above should also override any other button styles (including the primary button used in the "popular" plan)

![Screenshot 2020-11-27 at 12 48 42](https://user-images.githubusercontent.com/1083581/100446323-e509ce80-30ae-11eb-90e0-e9f7788f88e6.png)

## Context

This was never a requirement for the Plans Grid. We'll need to understand if this change should apply only to the Focused Launch, or if it should affect the Plans Grid component in all of its variations.

**Comment by ciampo** [comment]
Hey @ollierozdarz , we need your input on this one — do you mind reviewing this issue and provide some guidance? Thank you!

**Comment by ollierozdarz** [comment]
@ciampo I've updated the [Figma](https://www.figma.com/file/8SZljM999DKC7tWBlONFCR/Launch-Flow-All-users?node-id=1%3A93) file to include a selected state. You'll also notice I've edited the working on the 'Free plan' button label when it's not available from "Free Plan" to "Unavailable with domain" to make it more clear **why** it's not available (this was from some feedback I just received). Sorry if this is extra work to change! 

<img width="1435" alt="Screen Shot 2020-11-27 at 3 19 17 pm" src="https://user-images.githubusercontent.com/67085494/100414236-f5c53f00-30c4-11eb-8c96-a92c1dd3d180.png">
